### DISCUSSION TITLE
Barulho excessivo em ambientes de trabalho aberto

### DISCUSSION THREAD

**Discussion by faefelipe** [frontendbr/forum#1097] [Discussão]
Olá pessoal, estou trabalhando em ambiente aberto (nossos queridos open spaces), contendo em torno de 40 pessoas desenvolvedoras, porém este ambiente em determinados momentos, ocorre um barulho excessivo de conversas paralelas  dificultando a realização de qualquer tarefa cotidiana.

Com o agravante de que muitos fazem ligações via hangout com membros de equipe e clientes, normalmente alguem acaba falando mais alto ou encaminhando via Slack uma reclamação devido ao barulho.

Alguém já teve problemas parecidos? Tem alguma dica pra ajudar a lidar com esse problema?

**Comment by fakenickels** [comment]
Geralmente eu só ponho o velho e bom headphone JBL e boto uma música, se eu enjoar das músicas eu boto algum vídeo de barulho de chuva ou vento de 10 horas

**Comment by faefelipe** [comment]
@grsabreu É uma alternativa que utilizo, mas é complicado pois o problema (barulho do pessoal) continua existindo sabe.

**Comment by mateusmiguel** [comment]
Falamos disso semana passada aqui na empresa. Somos uns 70. Uma ideia que surgiu pra ajudar quando o volume fica muito alto é levantar a mão, daí quem te vê levanta também e por aí vai. Não é perfeito, mas ajuda um pouco.

  ¯\_(ツ)_/¯

**Comment by vsppedro** [comment]
Uma opção similar ao headphone, e que usei algumas vezes, é o abafador de ruído( e você pode ainda colocar um fone mais simples por dentro). Duvido que essa combinação permita que algum ruído lhe incomode. 

Agora para convencer a galera a falar mais baixo, só na base da conversa.   

Talvez simples avisos no local, como o que existem em bibliotecas, possam ajudar o pessoal a lembrar que ali também é necessário silêncio para que a concentração dos outros não seja interrompida. Tem gente que esqueçe...

Se pedidos educados não forem suficientes, talvez algo como penalizações sobre execesso de barulho sejam necessárias. Certas coisas só entram na cabeça das pessoas quando pesam no bolso. Exemplo, após dois avisos se o incômodo persistir, bota esse barulhento pra comprar café em pó pra todo mundo. 

Espero que a caminhada até a padaria faça esse cidadão refletir sobre como trabalhar em um ambiente aberto. 

OBS: A última sugestão seria tipo um manual de boas práticas, direitos e deveres, daqueles que participam do ambiente. Logicamente teria que ser debatido e aprovado por todos.

OBS2: essas são sugestões de alguém que nunca trabalhou em um open space. Aprecie com moderação...

**Comment by cezarsmpio** [comment]
Isso realmente é um problema... não só em open spaces, pois pra mim, qualquer lugar em que você trabalha é um open space a não ser se você trabalha isolado em uma sala. 

Nos meus últimos trabalhos todos trabalhavam juntos. Algumas empresas o "open space" era menor e outras muito grande. 

> Com o agravante de que muitos fazem ligações via hangout com membros de equipe e clientes.

A empresa deveria tomar providências sobre isso, talvez fornecer salas para esse tipo de reunião.

Acho que essas conversas excessivas sempre vão existir, até mesmos nós fazemos isso pois é mais rápido conversar ali cara a cara e resolver algo rápido ao invés de outros meios, porém se isso atrapalhar o ideal seria ter salas ou ir para um canto.

Já trabalhei em lugares que tinham poucas salas e a conversa paralela era complicado.

Eu como dev tento me desligar, colocar uma música (não precisa ser alta) e focar naquilo que estou fazendo. Podcasts me ajudam.

Enfim, acho que as empresas deveriam tomar providências e oferecer mais lugares para isso e também tentar mudar a cultura da empresa. Slack e emails estão aí justamente para evitar isso.

Acho que o jeito é também saber se adaptar :)

**Comment by caioincau** [comment]
Depois de mais de um ano pensando, decidi investir em um fone Bose com Noise Cancelling, melhor coisa que comprei no ano, o barulho continua, as vezes um ou outro grita e da um esporro, mas ao menos, resolvi o meu lado :)

**Comment by maclevison** [comment]
É um assunto complicado, pois envolve cultura da empresa e também muito bom senso das pessoas. Algumas pessoas tem um poder de foco muito grande, o que não é meu caso, e conseguem de boas se manter a produção em ambientes barulhentos.

Por diversas vezes fui questionado sobre minha produção diária e comecei a investigar o motivo de alguns dias serem bastantes produtivos e outros não, mas da mesma forma saia do trampo cansado mentalmente e com a sensação de que não fiz nada durante o dia, observei que o barulho da sala me consumia energia pois perdia o foco com as conversas e me esforçava demais pra voltar ao ritmo.

Me sugeriram fones com isolamento, mas nem sempre gosto de codificar com música, na maioria das vezes prefiro o silêncio total, e é esse o ponto chave do porque as noites são mais produtivas pra mim.

Hoje eu me desligo das redes sociais e mensageiros, uso os fones com Noise Cancelling sem tocar nada ou com música eletrônica, quando o barulho está sem controle procuro conversar com a galera pra dar uma moderada, ou uso protetor auricular industrial _(aqueles que operadores de trator usam)_.

Uma boa solução seria as empresas adotarem um manual de boas práticas e segui-lo.

:)

**Comment by caioincau** [comment]
Tbm uso o Noise Cancelling sem música, melhor coisa.

**Comment by shadowmaru** [comment]
Tenho fones com noise cancelling, mas não fica ideal sem música, e às vezes não quero ouvir música. Usar o fone por muito tempo também me dá desconforto, e às vezes até dor de cabeça.

Tem alguns momentos que percebo que as conversas ficam mais intensas: perto da hora do almoço e umas 18h. Mesmo as pessoas que saem depois já se sentem à vontade para bater papo.

Uma coisa que funcionou por um tempo foi um sino, que alguém tocava quando o barulho estava muito alto. Depois evoluiu para :bell: no Slack, mais silencioso. Mas caiu em desuso.

**Comment by kelciocajueiro** [comment]
Geralmente eu utilizo um fone intra-auricular que elimina qualquer som externo. Quando não to afim de ouvir música cantada coloco um instrumental mesmo e pá. Quando eu trabalhava em um ambiente aberto, todos se conheciam e tal então eu soltava um "EITA ZUADA DA P****". Também funcionava kkk

**Comment by felquis** [comment]
Eu fazia muito `CHIIIIIIIIIIIIIIIII` e todos paravam por uns 20 segundos e se conscientizavam, mas logo voltavam, mas era divertido.

A firma também buscou um andar em que usam carpet no chão, o que diminuiu o volume

Também surgiu a idea de usar abafadores acústicos nas partes e painéis no meio do escritório para evitar que o som de longe prolifere tanto.

**Comment by RatoX** [comment]
Isso me incomoda bastante, não o barulho mas sim as reclamações: `seres humanos fazem barulho` acostumem-se, não tem o que fazer. A maquina precisa girar e não vai ser por que um peão se incomoda com o calor que teremos que desligar as fornalhas. 

Se tu tem problemas de concentração por causa do barulho, acostume-se o mundo é barulhento e procure algum tratamento para lidar com isso, procure profissionais da área.

Se o barulho atrapalha literalmente o seu trabalho converse com superiores e bole um plano junto com eles: Melhor acabamento acústico nos escritórios, tele-trabalho, mais salas de reuniões e por ai vai.

Se tu se incomoda por que humanos conversam, sinto lhe informar mas a sociedade funciona assim. Nosso trabalho não é especial e não somos diferentes de ninguém, poucos trabalhos realmente precisam de um silencio absoluto mas geralmente todos precisam de concentração.

**Comment by faefelipe** [comment]
@RatoX, acho que você entendeu um pouco errado a intenção da issue, esta não foi criada como um pequeno desabafo de um desenvolvedor, mas sim de uma situação que está ocorrendo e atrapalhando o desenvolvimento de colegas de trabalho(dev) num contexto geral. 
Abri a issue com a intensão de coletar experiências de outras pessoas que podem ter vivenciado algo parecido e como resolveram (ou não) este problema sem rolar uma imposição ou xingamento. 
Já estamos conversando com o RH da empresa, afim de coletar informações das pessoas e tentar melhorar a cultura nesse quesito.

**Comment by RatoX** [comment]
@faefelipe nego eu te entendo  e meu comentário não foi pra tu, foi pra toda nossa área. 

Barulho é um problema, acontece, sempre aconteceu mas não tem jeito. Temos que lidar com isso de maneira individual, se não for algo danoso a saude ou algo extremamente fora do comum ( alguém ouvindo som alto no escritório ) a gente não pode pedi as pessoas que parem de se comunicar.

**Comment by faefelipe** [comment]
@cezarlz Esqueci de comentar que dentro da empresa existem alguns locais mais isolados para uma reunião | Call com cliente, porém o barulho nem sempre é relacionado aos grupos de pessoas conversando ocasionando na Call, mas o restante do pessoal. E nem toda equipe possui um notebook, por exemplo para levar aos espaços mais "silenciosos" da empresa.

**Comment by RatoX** [comment]
@faefelipe tow ligado mas empresa de escritórios geralmente é algo bem dinamico e imprevisivel:

- Vendas: as vezes tu tá trampando e o cliente te liga naquela hora/minuto/segundo pra saber de algum detalhe e tu não tem nem tempo de marcar uma sala de reunião
- Dev: tu tem uma duvida e a pessoa que pode responder tá atrás de tu
- Limpeza: é o seu horário e dia de passar aspirador como combinado
- Gerente: tu encontrou aquela pessoa que tava saindo de férias naquele exato momento e precisa saber onde ela deixou a chave do armario 
- Liderança Técnica: Seu time tá com duvidas sobre uma integração e a reunião com a empresa é daqui a 30 minutos e tu tem que fazer uma mini-palestra de 10 minutos para 5 pessoas explicando como funciona
- Porteiro: Chegou a correspondecia na hora e ele teve que subir para entregar pq tava chegando o horário dele ir embora já que ele chega as 05 da manhã para abrir o prédio, não tinha ninguém e ele teve que ir chamando os nomes em voz alta


e  por ai vai

Existem diversas coisas acontecendo e nós, desenvolvedores, não somos especiais, o barulho faz parte do dia-a-dia.

**Comment by diasdev** [comment]
@RatoX pessoas conversam, nem por isso alguém deveria ser obrigado a escutar a conversa de dezenas de pessoas todos os dias, o dia todo. E tampouco isso justifica passar o dia todo no trabalho conversando em voz alta sobre filme, videogame, fim de semana, futebol, etc etc etc. É uma atitude que não ajuda essa de "a gente é assim mesmo e os incomodados que se retirem". Ambiente profissional na área de desenvolvimento deve ser propício à atividade intelectual, e tagarelice o dia todo não proporciona isso.

**Comment by RatoX** [comment]
> E tampouco isso justifica passar o dia todo no trabalho conversando em voz alta sobre filme, videogame, fim de semana, futebol, etc etc etc.

Sim, tem razão se pessoas passam o dia assim no ambiente de trabalho e se vem atralhando isso tem que ser tratado individualmente. 

> Ambiente profissional na área de desenvolvimento deve ser propício à atividade intelectual, e tagarelice o dia todo não proporciona isso.

É ai é que entra meu ponto, nem tudo é tagarelice como citei acima 1001 coisas podem acontecer e que fazem parte do dia-a-dia, geralmente elas ocorrem e não tem o que fazer. 

> propício à atividade intelectual

E sobre esse ponto eu concordo mas também ele tem que ser propício para que o trabalho flua, se isso requer que eu tenha que virar pro lado e perguntar algo pra alguém é tão valido quanto.

E a gente tem que lembrar que sem uma boa acústica 50 pessoas sussurando vai ser barulhento como 2 pessoas falando "alto".

**Comment by woliveiras** [comment]
Acredito que entra muito a questão da colaboração geral (afinal é todo mundo um time).

Se o problema é o barulho enquanto alguém está em uma ligação com clientes ou com alguém que está remoto, por exemplo, vale a pena a pessoa dar um toque, como levantar o braço pra sinalizar que o que está fazendo está sendo atrapalhado pelo barulho e o time, provavelmente, vai entender, vai esperar a ligação acabar de depois a vida volta ao normal. - se não atenderem a próxima vez que levantar o braço tem que ser com uma faca na mão :hocho: 

Se o problema é eu, como dev, não conseguir me concentrar pelo barulho, vale muito a pena eu buscar alternativas para que eu consiga essa concentração. Afinal, a não ser que seja uma deficiência minha, não seria justo limitar a comunicação de todo mundo por conta de uma característica pessoal.

O que eu faço quando preciso me concentrar é colocar o fone e ouvir um som maneiro. E nem é um desses fones master tech que a galera comentou.

Normalmente as pessoas já nem nos chamam se estamos trabalhando com fone no ouvido.

Temos que fazer um levantamento pessoal:

- eu estou incomodado por algo que é característica minha?
- eu tenho problema de atenção que poderia ser tratado por um atendimento especializado?
- eu consigo colaboração das pessoas se eu avisar que estou em uma ligação?
- eu não estou me achando mais importante do que as outras pessoas e quero o ambiente silencioso pra escrever minhas maravilhas aqui?

Porque limitar a comunicação dentro de um ambiente onde nós conseguimos a liberdade (andar com a roupa que quisermos, falar na gíria, etc.) é querer voltar pro ambiente corp onde cada pessoa ficava numa baia e a comunicação era via email, né?

**Comment by RatoX** [comment]
@woliveiras ótimo pontos!

**Comment by aputhin** [comment]
Bom, eu sou colega de empresa do @faefelipe e posso garantir que não é uma questão "pessoal". O barulho é um problema do coletivo no nosso ambiente de trabalho, especificamente, e é trazido pra discussão esporadicamente. Se fosse 1 ou 2 reclamando acho que a issue dele seria "qual o melhor fone pra abafar ruído", enfim.

Acho mega problemático essa abordagem de que o coletivo é sagrado (ou "os incomodados que se retirem") por que isso é mega zona de conforto e não ajuda nada na discussão de como melhorar a cultura, que é a intenção da issue.

Entendo que exista um receio de não "limitar a colaboração" no ambiente de trabalho, mas com todo o aparato tecnológico e social que temos hoje em dia, impossível que a "colaboração" dependa de ficar falando alto e criar volume. O @diasdev foi dos poucos que entendeu por que trouxemos isso: o ambiente de trabalho não precisa necessariamente comportar tudo, ser uma anarquia onde todo mundo trabalha junto. Isso ao meu ver dá uma ilusão de produtividade e só, por que no fundo a nossa atividade exige bastante concentração. 

Não precisa colocar todo mundo em um cubículo, mas dar um espaço confortável onde todo mundo possa fluir sem precisar entrar numa bolha quando quiser ficar quieto. O aspecto social é importantíssimo sim, mas pode ser feito respeitando o espaço do outro.

**Comment by RatoX** [comment]
"O aspecto social é importantíssimo sim, mas pode ser feito respeitando o espaço do outro."
@aputhin concordo 100%

Eu acho que me expressei mal quando eu falei isso: "Se tu tem problemas de concentração por causa do barulho, acostume-se o mundo é barulhento e procure algum tratamento para lidar com isso, procure profissionais da área." e o @woliveiras trouxe bem mais detalhado.

Meu maior ponto é que um escritório é barulhento na sua dinamica e não tem o que ser feito sobre isso, existem focos de barulhos que não deveriam existir: "Pessoas tagalerando sobre uma festa na noite passada" mas existe toda uma outra dinamica que eu já listei aqui que não vamos resolver mandando todos ficarem calados ou pedindo bom senso.

**Comment by RatoX** [comment]
E o meu ponto principal sobre essa issue é que nós, desenvolvedores, costumamos a nos achar especiais e que precisamos de mais "regalias" que os demais funcionarios da empresa. Sendo que sem um trabalho coletivo nada avança.

**Comment by aputhin** [comment]
@RatoX não vejo como uma questão de "nos achar especiais" nem vejo silêncio como regalia. Existem atividades e atividades. Todas essas dinâmicas que tu colocou, se organizar direitinho, ninguém pisa no calo de ninguém. Não é por que um precisa de barulho que o outro é obrigado a escutar. Querer impor um nível de aceitação mais alto a longo prazo cria uma cultura onde todo mundo só quer saber do seu e ignora o espaço do outro.

Novamente, isso que tu colocou é uma forma de justificar não discutir o problema cultural do barulho, mas o que nós estamos fazendo / pretendemos é justamente isso. Tentar evitar a discussão não ajuda muito, honestamente. Entendo a tua opinião, mas já passamos da parte de validar se é problema pessoal ou cultural.

**Comment by RatoX** [comment]
@aputhin

Entendo, sendo que o caso de vocês não é pessoal e tem um monte de gente sendo impactado pelo barulho excessivo, vocês já tentaram fazer uma lista pontuando durante uma semana quais são os focus dos barulhos?

As vezes estamos falando apenas de acústica que pode ser resolvida com uns 50 mil reais e uma boa engnheira de som. ( que para maioria das empresas é um valor irrelevante se estiver trazendo demora nas entregas )

**Comment by RatoX** [comment]
E sobre não discuti problema cultural do barulho, não é questão de colocar a barra alta ou baixa é questão de aceitar que são ambientes barulhentes. Se tiver alguém passando dos “limites” que se trate individualmente, mas que a cultural do: Fique calado pq estou trabalhando, não impere também.

**Comment by RatoX** [comment]
Tenho um bom exemplo de resolução disso, algumas empresas criam a tal da sala silenciosa, geralmente um espaço próprio onde é proibido todo tipo de conversa ou barulho. 

Pode ser uma

**Comment by diasdev** [comment]
Vejo esses dois pontos (colaboração e liberdade) da seguinte forma:

Colaboração não é um princípio universal e total. Se o colega A está falando com o colega B sobre o projeto X, eu não tenho por que estar escutando essa conversa. Nem sobre o projeto Y nem o Z, e geralmente está ocorrendo tudo ao mesmo tempo. Nosso desempenho depende de resolver problemas, muitas vezes complexos, e isso exige longos períodos de concentração. O ambiente de trabalho deveria ser tratado como biblioteca (não sei se adiantaria, já que nosso povo vai pra biblioteca conversar com os amigos e falar no telefone).

A liberdade é outro ponto que sempre à tona. Acontece que se você criar um clima de todos podem tudo, logo a liberdade dos mais abusados vai se expandir e esmagar a liberdade dos mais moderados. Vira selva, onde reina a lei do mais forte. É pra conviver em sociedade sem um se sobrepor ao outro que criamos incontáveis códigos de conduta e comportamento, restrições à liberdade, coerções externas mesmo, pra que todos possam conviver. Esse ambiente de startup que veio com os millenials ignora tudo isso.

As empresas geralmente implementam esses espaços com a desculpa da colaboração mas é quase sempre pra poupar dinheiro, já que montar esses telemarketings ocupa muito menos espaço. O que dá pra fazer, muitas vezes, é separar os times, assim os membros só escutam o que é relevante à sua equipe. E criar também ambientes comuns para quando é preciso juntar mais gente (salas de reunião, espaços abertos, etc).

Quem tem mais dinheiro, como a Microsoft, cria vários ambientes diferentes, cada um pra uma necessidade. Na MS os engenheiros têm desde salas privadas à ambientes abertos pra discussão. O segredo é ajustar isso ao orçamento da companhia.

Por aqui muito se resolveria se o brasileiro não fosse tão extrovertido e tagarela. Open plan offices surgiram e funcionam na Alemanha por um motivo. hehehe

**Comment by diasdev** [comment]
@RatoX talvez não um "fique calado", mas certamente um "faça menos barulho porque estou trabalhando" é muito mais racional e aceitável do que "estou conversando, então vá trabalhar em outro lugar".

**Comment by RatoX** [comment]
> As empresas geralmente implementam esses espaços com a desculpa da colaboração mas é quase sempre pra poupar dinheiro, já que montar esses telemarketings ocupa muito menos espaço.

Concordo 100%

> é separar os times, assim os membros só escutam o que é relevante à sua equipe.

Isso vai diminuir bastante mas não vai zerar afinal pra coisas funcionarem vai precisar de integração.

> "faça menos barulho porque estou trabalhando" é muito mais racional e aceitável do que "estou conversando, então vá trabalhar em outro lugar".

E quando a "conversa" é também trabalho?

**Comment by RatoX** [comment]
Acho que já trouxes meus pontos e eu entendi o de vocês. 

Ainda acredito que na maioria dos casos, de problema de concentração devido ao barulho de um escritório, é um problema individual e que em algumas ocasiões pode até ser preciso ajuda médica, porém também concordo com vocês que se eu tiver no escritório jogando papo pro ar eu devo ficar esperto pra não atrapalhar meus colegas e tentar não ser um cuzão.

Meu último ponto é sobre um pedaço dessa frase:  "Por aqui muito se resolveria se o brasileiro não fosse tão extrovertido e tagarela." ela é engraçada por que somos brasileiros mas me lembra algumas situações que passei aqui em São Paulo e em Porto Alegre, em que pessoas me chamavam de *baiano escandaloso* por causa da forma em que eu me comunicava, que são formas diferentes. E até quando eu morei em Recife por um tempo, ouvi alguns norte-americanos soltando o mesmo tipo comentários sobre alguns Indianos. Já vi contemporaneos tendo que ir, obrigados, no fonoaudiólogo por que o xiado "atrapalhava a comunicação". Não vou me extender nesse assunto e tenho certeza que não foi esse o ponto da frase e sim eu concordo, nós brasileiros somos tagaleras! ahahahahahhaa

**Comment by designernatan** [comment]
meus 2 cents: na caelum antigamente quando o barulho incomodava, pediamos silencio

problema é que pessoas novas ou mais tímidas não ficavam a vonts pra gritar pedindo silencio na hora ~~da feira~~ que acontecia

aí resolvemos compramos sinetinhas daquelas de hotel, distribuimos nas mesas, aí até que ajuda bem hoje em dia :)

**Comment by bernardodiasc** [comment]
O ambiente de trabalho de um desenvolvedor de sistemas não deveria ser diferente de uma biblioteca...

**Comment by ghost** [comment]
Talvez falte nas pessoas o famoso " semancol ". Não me incomodo do colega ao lado estar conversando sobre qualquer coisa com alguém, desde que o volume da voz seja baixo e não atrapalhe os demais.

**Comment by caioincau** [comment]
Pois é, tudo em moderação, não tem problemas, mas tem casos extremos.

**Comment by jhonnymoreira** [comment]
Trabalhava ao lado do pessoal de vendas na primeira empresa que participei, pra diminuir qualquer som, eu comprei uns earbuds da Sennheiser. Porém, com o passar do tempo eu descobri uns modelos da Audio Technica com Quiet Point, que é um dispositivo de switch que isola todo o som ao redor. Depois que tive um desses, não larguei mais. Aconselho forte!
